test/python/transpiler/test_vf2_post_layout.py

```diff
@@ -239,6 +239,39 @@ def test_2q_circuit_5q_backend_controlflow(self):
         self.assertLayout(dag, cmap, pass_.property_set)
         self.assertNotEqual(pass_.property_set["post_layout"], initial_layout)
 
+    def test_2q_circuit_5q_backend_max_trials(self):
+        """A simple example, without considering the direction
+          0 - 1
+        qr1 - qr0
+        """
+        max_trials = 11
+        backend = FakeYorktown()
+
+        qr = QuantumRegister(2, "qr")
+        circuit = QuantumCircuit(qr)
+        circuit.cx(qr[1], qr[0])  # qr1 -> qr0
+        tqc = transpile(circuit, backend, layout_method="dense")
+        initial_layout = tqc._layout
+        dag = circuit_to_dag(tqc)
+        cmap = CouplingMap(backend.configuration().coupling_map)
+        props = backend.properties()
+        pass_ = VF2PostLayout(
+            coupling_map=cmap, properties=props, seed=self.seed, max_trials=max_trials
+        )
+
+        with self.assertLogs(
+            "qiskit.transpiler.passes.layout.vf2_post_layout", level="DEBUG"
+        ) as cm:
+            pass_.run(dag)
+        self.assertIn(
+            f"DEBUG:qiskit.transpiler.passes.layout.vf2_post_layout:Trial {max_trials} "
```

Comment:
The test is failing because there is a space missing in the assertion (at least according to the logs: https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=45472&view=logs&j=e4ea27c2-3a16-5b32-7be8-0bf30fe9e828&t=793d4f07-8490-552a-798f-6a5657042d16&l=24059 ). But looking here there is a space. I'm not sure exactly what's causing this.

Reply:
Oh, you fixed it in: https://github.com/Qiskit/qiskit-terra/pull/9963/commits/ccdbab0850f9e1c7c9fc4fbd4732882479f89b28 between when I looked at the test failure and left the comment